nomad/job_endpoint_hook_implicit_identities.go

```diff
@@ -10,8 +10,8 @@ import (
 )
 
 const (
-	consulServiceIdentityNamePrefix = "consul-service"
-	consulTaskIdentityNamePrefix    = "consul"
+	ConsulServiceIdentityNamePrefix = "consul-service"
+	ConsulTaskIdentityNamePrefix    = "consul"
```

Comment:
Since these values are shared across many different places I placed the Vault prefix constant in the `structs` package:
https://github.com/hashicorp/nomad/blob/cecd9b04729e63839428cfa67fdc983ed2182481/nomad/structs/workload_id.go#L23-L25